synth.py

```diff
@@ -21,27 +21,32 @@
 
 logging.basicConfig(level=logging.DEBUG)
 
-gapic = gcp.GAPICGenerator()
+gapic = gcp.GAPICMicrogenerator()
 common_templates = gcp.CommonTemplates()
 
 versions = ['v1', 'v1beta1']
 
 for version in versions:
-    library = gapic.node_library('texttospeech', version)
+    library = gapic.typescript_library(
+        'texttospeech', 
+        generator_args={
+            "grpc-service-config": f"google/cloud/texttospeech/{version}/texttospeech_grpc_service_config.json"
+            },
+        proto_path=f'/google/cloud/texttospeech/{version}',
+        version=version)
 
     # skip index, protos, package.json, and README.md
     s.copy(
         library,
-        excludes=['package.json', 'README.md', 'src/index.js'],
+        excludes=['package.json', 'README.md', 'src/index.ts'],
     )
 
-templates = common_templates.node_library()
+templates = common_templates.node_library(source_location='build/src')
 s.copy(templates)
 
-# Fix dead link
-s.replace('src/v1/doc/google/cloud/texttospeech/v1/doc_cloud_tts.js',
-        "\(https:[\s\*]+(.*)\)",
-        r"(https:\1)")
+# Fix system tests
+# TODO: must be a feature of pack-n-play
```

Comment:
What's missing here?  Do we need a tracking bug over in pack-n-play?

Reply:
We have a tracking issue here https://github.com/googleapis/gapic-generator-typescript/issues/134 - undecided if it's gonna be in `pack-n-play` or an option for the generator. The problem is that the generator does not know the package name 😿